core/domain/user_services.py

```diff
@@ -2528,39 +2568,43 @@ def remove_contribution_reviewer(user_id):
         user_contribution_rights_model.delete()
 
 
-def get_contribution_reviewer_usernames(review_category, language_code=None):
-    """Returns a list of usernames of users who has rights to review item of
-    given review category.
+def get_contributor_usernames(category, language_code=None):
+    """Returns a list of usernames of users who has contribution rights of given
+    category.
 
     Args:
-        review_category: str. The review category to find the list of reviewers
+        category: str. The review category to find the list of reviewers
             for.
         language_code: None|str. The language code for translation or voiceover
             review category.
 
     Returns:
         list(str). A list of usernames.
     """
-    reviewer_ids = []
-    if review_category == constants.REVIEW_CATEGORY_TRANSLATION:
-        reviewer_ids = (
+    user_ids = []
+    if category == constants.REVIEW_CATEGORY_TRANSLATION:
+        user_ids = (
             user_models.UserContributionRightsModel
             .get_translation_reviewer_user_ids(language_code))
-    elif review_category == constants.REVIEW_CATEGORY_VOICEOVER:
-        reviewer_ids = (
+    elif category == constants.REVIEW_CATEGORY_VOICEOVER:
+        user_ids = (
             user_models.UserContributionRightsModel
             .get_voiceover_reviewer_user_ids(language_code))
-    elif review_category == constants.REVIEW_CATEGORY_QUESTION:
+    elif category == constants.REVIEW_CATEGORY_QUESTION:
         if language_code is not None:
             raise Exception('Expected language_code to be None, found: %s' % (
                 language_code))
-        reviewer_ids = (
+        user_ids = (
             user_models.UserContributionRightsModel
             .get_question_reviewer_user_ids())
+    elif category == constants.SUBMIT_QUESTION_CONTRIBUTION_RIGHT_CATEGORY:
```

Comment:
It seems like we're changing the naming style for the constant here (e.g. ``REVIEW_CATEGORY_QUESTION`` vs. ``SUBMIT_QUESTION_CONTRIBUTION_RIGHT_CATEGORIY``). Can we make them all consistent?